### DISCUSSION TITLE
Sophos detected mpv.com as malware

### DISCUSSION THREAD

**Discussion by Marioux37** [rocksdanister/lively#1258] [Q&A]
Hi,

The latest version of the app (got updated recently) is detected by Sophos Endpoint as mallware

Detection nameMal/EncPk-ANY

SHA 256bc050228722cdecabebef58be9b7e1824cdcff7c5daf07ea3ca0a563d2cca27d 
PathC:\Program Files\WindowsApps\12030rocksdanister.LivelyWallpaper_1.0.119.0_x86__97hta09mmv6hy\Build\Plugins\Mpv\mpv.com 
![Web capture_9-6-2022_115253_central sophos com](https://user-images.githubusercontent.com/91732630/172806952-5e4e46b1-7bd5-4a27-bb58-65b2ab065bf4.jpeg)

**Comment by rocksdanister** [comment]
duplicate: https://github.com/rocksdanister/lively/issues/358

You are free to submit it for re-evaluation to Sophos:
https://www.virustotal.com/gui/file/bc050228722cdecabebef58be9b7e1824cdcff7c5daf07ea3ca0a563d2cca27d

**Reply by Marioux37** [reply]
I assumed it was a false positive. Just tried to help. Thank you for your work. App's amazing.

**Reply by rocksdanister** [reply]
I understand, its just not practical for me to contact every AV vendors for each release.
